src/Blocks/BlockPiston.h

```diff
@@ -79,6 +81,11 @@ class cBlockPistonHandler :
 		}
 	}
 
+	/** This method converts the magic piston metadata into a direction vector.
+		This vector has a length of 1 and points into the direction, in which the piston will extend.
+	*/
```

Comment:
-> `/** Converts piston block's metadata into a unit vector representing the direction in which the piston will extend. */